Packs/CarbonBlackDefense/Classifiers/classifier-Carbon_Black_Endpoint_Standard_5_9_9.json

```diff
@@ -5,11 +5,6 @@
     "feed": false,
     "isDefault": false,
     "id": "Carbon Black Endpoint Standard",
-    "keyTypeMap": {
-        "High": "Carbon Black Endpoint Standard",
-        "Low": "Carbon Black Endpoint Standard",
-        "Normal": "Carbon Black Endpoint Standard"
-    },
```

Comment:
why did you remove this?

Reply:
![Screen Shot 2021-05-09 at 15 28 08](https://user-images.githubusercontent.com/78307768/117572357-38129380-b0db-11eb-81b7-df130b83f73a.png)